src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -104,7 +112,7 @@ public User getUser(UUID id) {
 
   /**
    * Add a new user to the current set of users known to the application. This should only be called
-   * to add a new user, not to update an existing user.
+   * * to add a new user, not to update an existing user.
```

Comment:
extra star

Reply:
Oops, I merged it too early, I will include those changes in the latest PR

Reply:
fixed